### ISSUE TITLE
Add version tags

### ISSUE THREAD

**Issue by AnHeuermann** [modelica/fmi-cross-check#83]
### Problem
When running tests with my FMI tool using the master branch of this repository and something fails I can't be 100% sure if the error is on my side or there was a new commit wich did break something.
E.g. there are problems like #69 
But of cours I don't want do define a commit as "correct" and start to only test with that version.

Also having constantly chaing number of tests makes it hard to follow why my results are getting better or worse.

### Suggestion
Add some version tags on some specific commits where (at least mostly) everything is correct.
To find a precise criteria for this is probably hard, but still better than trying to keep up with a ever chaning master.

Or is there a different approach I could be using for this one?
Maybee rejecting every commit that is failing your tests on circleci?
Not that it's matter that much since my tool has to many basic problems, but I hope I'm not the only one to use this repo.

**Comment by t-sommer** [comment]
All FMUs and results on the master branch should be either compliant or marked as not compliant.
If not, please open an issue and describe the problem.
